knowledge-graph/src/main/scala/io/renku/knowledgegraph/projects/files/lineage/EdgesFinder.scala

```diff
@@ -71,19 +71,22 @@ private class EdgesFinderImpl[F[_]: Async: Logger: SparqlQueryTimeRecorder](
     Prefixes.of(prov -> "prov", renku -> "renku", schema -> "schema"),
     s"""|SELECT DISTINCT ?activity ?date ?sourceEntityLocation ?targetEntityLocation
         |WHERE {
-        |   ${projectMemberFilterQuery(ResourceId(path)(renkuUrl).showAs[RdfResource])(maybeUser)}
-        |   ?activity a prov:Activity;
-        |             ^renku:hasActivity ${ResourceId(path)(renkuUrl).showAs[RdfResource]};
-        |             prov:startedAtTime ?date;
-        |             renku:parameter ?paramValue.
-        |   ?paramValue schema:valueReference ?parameter.
-        |   OPTIONAL {
-        | 	  ?parameter a renku:CommandInput.
-        |     ?paramValue schema:value ?sourceEntityLocation.
-        |   }
-        |   OPTIONAL {
-        | 	  ?parameter a renku:CommandOutput.
-        |     ?paramValue schema:value ?targetEntityLocation.
+        |   BIND (${ResourceId(path)(renkuUrl).showAs[RdfResource]} AS ?projectId)
+        |   Graph ?projectId {
+        |     ${projectMemberFilterQuery(ResourceId(path)(renkuUrl).showAs[RdfResource])(maybeUser)}
+        |     ?activity a prov:Activity;
+        |               ^renku:hasActivity ${ResourceId(path)(renkuUrl).showAs[RdfResource]};
```

Comment:
I guess as we bound `ResourceId(path)(renkuUrl).showAs[RdfResource]` to `?projectId` we could use the `?projectId` here instead of creating the `ResourceId` again.